applications/butler/templates/ingress-anonymous.yaml

```diff
@@ -20,25 +20,26 @@ template:
       - host: {{ required "global.host must be set" .Values.global.host | quote }}
         http:
           paths:
+            {{- range $repositoryLabel := .Values.config.repositoryLabels }}
             # For direct end-user use of the Butler client library, the
             # Butler() convenience constructor must be able to load a
             # configuration file via unauthenticated HTTP.  This exists for
             # compatibility with the way Butler instances were configured prior
             # to the existence of the Butler server -- they are passed the URI
             # for a repository root on the filesystem or HTTP, from which a
             # configuration file is loaded.
```

Comment:
Very minor nit, but I would put the comment outside of the range, since there's no need to repeat it for each repository.